services/shhext/chat/protocol.go

```diff
@@ -124,8 +124,13 @@ func (p *ProtocolService) GetPublicBundle(theirIdentityKey *ecdsa.PublicKey) (*B
 	return p.encryption.GetPublicBundle(theirIdentityKey)
 }
 
+// ConfirmMessagesProcessed confirms and deletes message keys for the given messages
+func (p *ProtocolService) ConfirmMessagesProcessed(messageIDs [][]byte) error {
```

Comment:
I can't find a reference to this method anywhere. What does call it?

Reply:
:man_facepalming: forgot to wire things up, well spotted